CHANGELOG.rst

```diff
@@ -5,6 +5,21 @@ Changelog
 `Unreleased <https://github.com/Ouranosinc/xsdba>`_ (latest)
 ------------------------------------------------------------
 
+Contributors:
+
+Changes
+^^^^^^^
+* No change.
+
+Fixes
+^^^^^
+* No change.
```

Comment:
I think I need to address this in the cookiecutter. It would be good to not show the `unreleased` block when we do a `bump-my-version bump release`. Let's keep it for now though.